commerce_coordinator/settings/base.py

```diff
@@ -348,7 +348,11 @@ def root(*path_fragments):
 # Needed to link to the payment micro-frontend
 PAYMENT_MICROFRONTEND_URL = 'replace-me'
 
-# Coordinator filters should NEVER be allowed to fail silently
+# Checkout view urls
+COMMERCETOOLS_FRONTEND_URL = 'https://checkout.edx.org'
```

Comment:
We should add values for local development here. Prod and stage config should be overridden from here: [edx-internal link](https://github.com/edx/edx-internal/tree/4b294b211801c57979c820795eea4e555b2597fd/argocd/applications/commerce-coordinator)